provisioning/service/tests/Config/BulkEnrollmentOperationErrorTests.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Newtonsoft.Json;
+
+namespace Microsoft.Azure.Devices.Provisioning.Service.Test
+{
+    [TestClass]
+    public class BulkEnrollmentOperationErrorTests
+    {
+        private const string REGISTRATION_ID = "valid-registration-id";
+        private const int ERROR_CODE = 400;
+        private const string ERROR_STATUS = "Bad message format exception.";
+
+        /* SRS_BULK_ENROLLMENT_OPERATION_ERRO_21_001: [The constructor shall throws ProvisioningServiceClientException if the 
+                                            provided registrationId is null, empty, or invalid.] */
+        [TestMethod]
+        [TestCategory("DevService")]
+        public void BulkEnrollmentOperationError_Constructor_ThrowsOnInvalidParameters()
+        {
+            // arrange
+            string nonRegistrationId = "{ \"errorCode\":" + ERROR_CODE + ", \"errorStatus\":\"" + ERROR_STATUS + "\" }";
```

Comment:
Nit - nicer and probably faster if you use `$"\"errorCode\":{ErrorCode}"`, etc

Reply:
I did that, but I'm not sure that it was a good idea because this is a JSON string, and the C# `{ }` became confused together with the JSON `{ }`